installation/linux.md

```diff
@@ -354,20 +354,77 @@ sudo rm -rf /opt/openhab2/
+
+* Package repository based installation:
+
+  ```shell
+  tail -f /var/log/openhab2/openhab.log -f /var/log/openhab2/events.log
+  ```
+
+* Manual installation:
+
+  ```shell
+  tail -f /opt/openhab2/userdata/logs/openhab.log -f /opt/openhab2/userdata/logs/events.log
+  ```
+
+You could even set up an SSH configuration (in Putty or similar) to automatically connect and execute the commands every time you start working on your setup.
+
+With openHAB 2 you can also [use the Karaf console]({{base}}/administration/logging.html#karaf-console) to have a colored glance at the logging information.
+
+## Recommended Additional Setup Steps
+
+The following is not directly related to the openHAB installation but rather recommended on a openHAB system.
+The need for these and the exact implementation on a specific system might differ from user to user.
+
+### Privileges for Common Peripherals
+
+An openHAB setup will often rely on hardware like a modem, tranceiver or adapter to interface with home automation hardware.
+Examples are a Z-Wave, Enocean or RXFcom USB Stick or a Raspberry Pi addon board connected to the serial port on its GPIOs.
+In order to allow openHAB to communicate with additional peripherals, it has to be added to corresponding Linux groups.
+The following example shows how to add Linux user `openhab` to the often needed groups `dialout` and `tty`.
+Additional groups may be needed, depending on your hardware and software setup.
 
-|                                  | Repository Installation  | Manual Installation (according to [guide](#manual-installation))  |
-|:--------------------------------:|:-------------------------|:------------------------------------------------------------------|
-| openHAB application              | `/usr/share/openhab2`    | `/opt/openhab2`                                                   |
-| Site configuration               | `/etc/openhab2`          | `/opt/openhab2/conf`                                              |
-| Log files                        | `/var/log/openhab2`      | `/opt/openhab2/userdata/logs`                                     |
-| Userdata like rrd4j databases    | `/var/lib/openhab2`      | `/opt/openhab2/userdata`                                          |
-| Service configuration            | `/etc/default/openhab2`  | (not preconfigured)                                               |
+```shell
+sudo useradd openhab dialout
+sudo useradd openhab tty
+```
 
-## Network Sharing
+Additionally it's needed to allow the java environment to access the serial port of the connected peripheral.
+Therefor the following setting has to be added/adapted on your system in either `/etc/default/openhab2` or `/usr/share/openhab2/runtime/bin/setenv`:
```

Comment:
Aaah I knew this was going to haunt me.

I'm a bit annoyed by that topic. I do not own any of these devices and asked the "community" to help out, no reaction - [here](https://github.com/ThomDietrich/openhabian/issues/33), or [here](https://github.com/ThomDietrich/openhabian/issues/37), or [here](https://community.openhab.org/t/openhabian-hassle-free-rpi-image/13379/262?u=thomdietrich) ... :sweat_smile: 

Anyhow. You are of course right. Judging by the logic inside setenv we can safely only recommend /etc/default/openhab2

Thanks for proof reading ;)